### PR TITLE
projectile-dir-files-external fix for frozen windows emacs

### PR DESCRIPTION
**Pull request by iamdionysus**
fix the issue https://github.com/bbatsov/projectile/issues/778

### CODE REVIEW

**Comment by bbatsov** [comment]
Looks good. Just mention the fixed issue in the commit message `[Fix #778] proje....` and in the changelog.

File: projectile.el [hunk 1]
```diff
@@ -733,12 +733,10 @@ Files are returned as relative paths to the project root."
 
 (defun projectile-dir-files-external (root directory)
   "Get the files for ROOT under DIRECTORY using external tools."
-  (let ((default-directory directory)
-        (files-list nil))
-    (setq files-list (-map (lambda (f)
-                             (file-relative-name (expand-file-name f directory) root))
```
Comment (bbatsov) [inline projectile.el]: We should probably check the code for more places when we can apply this optimization.

**Comment by iamdionysus** [comment]
Changed commit message to follow the convention.

**Comment by bbatsov** [comment]
:+1: Thanks!

**Pull request merged**